README.md

```diff
@@ -56,11 +56,25 @@ _Note_: To contribute, send a pull request to this repository. Note that this re
 ### Development Platform
 
 - [ONNX Runtime](https://onnxruntime.ai/): Platform agnostic model runtime to use ML models.
+- [TensorFlow](https://www.tensorflow.org/): An end-to-end open source platform for machine learning.
+- [PyTorch](https://pytorch.org/): An open source machine learning framework that accelerates the path from research prototyping to production deployment.
+- [PyTorch Lightning](https://github.com/Lightning-AI/pytorch-lightning) Pretrain, finetune and deploy AI models on multiple GPUs, TPUs with zero code changes.
+- [Scikit-learn](https://scikit-learn.org/stable/): A simple and efficient tools for predictive data analysis.
```

Comment:
I think this is not of use to developers, the idea of this repository is that the devs should just treat models as APIs and not have to train any model, and in sklearn this is not really possible.